test/slugid_test.js

```diff
@@ -0,0 +1,89 @@
+import { expect } from 'chai';
+import { nice, v4 } from '../src';
+
+/**
+ * `spreader` runs a test against the `generator` function, to check that
+ * when calling it 64*40 times, the range of characters per string position it
+ * returns matches the array `actual`, where each entry in `expected` is a
+ * string of all possible characters that should appear in that position in the
+ * string, at least once in the sample of 64*40 responses from the `generator`
+ * function */
+const spreader = (generator) => {
```

Comment:
wow, this is cool

Reply:
I can't take credit for this, I think this was all @jonasfj (ripped and modified from `slugid`).